new-readme.md

```diff
@@ -144,7 +144,7 @@ You can develop and run **reSolve** applications on **Linux**, **Mac** and **Win
 
 The minimum supported **NodeJS** version is **6.0.0**.
 
-We recommend using [**npx**](https://www.npmjs.com/package/npx) or [**yarn**](https://yarnpkg.com/lang/en/) instead of **npm**<!--, because ??? -->.
+We recommend using [**npx**](https://www.npmjs.com/package/npx) or [**yarn**](https://yarnpkg.com/lang/en/) instead of **npm** because these are faster and more convenient **npm** alternatives.
```

Comment:
`npx` is not a `npm` alternative!

`npx create-resolve-app` === `node ./node_modules/create-resolve-app/bin/index.js`

Reply:
It's an alternative to the `npm` **command**, not **tool**. I can add a word **command** there.